### PR TITLE
takeaway LSB for running arm thumb mode binary with qdb in qltool

### PR DESCRIPTION
**Pull request by ucgJhe**
as title.

### CODE REVIEW

File: qltool [hunk 1]
```diff
@@ -234,7 +234,7 @@ if __name__ == '__main__':
         # attach Qdb at entry point
         if options.qdb == True:
             from qiling.debugger.qdb import Qdb
-            ql.hook_address(Qdb.attach, ql.loader.entry_point)
+            ql.hook_address(Qdb.attach, ql.loader.entry_point & 0xfffffffe)
```
Comment (xwings) [inline qltool:237]: for arm / thumb only or all arch ?

Comment (ucgJhe) [reply qltool:237]: qdb is now working with MIPS and ARM, since MIPS is aligned by 4 byte its okay to both arch.

Comment (xwings) [reply qltool:237]: make it properly and be ready to x86 too :)

**Comment by xwings** [comment]
thanks !

**Pull request merged**